src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -204,29 +184,7 @@ class SidebarLinks extends React.PureComponent {
                             <Icon src={Expensicons.MagnifyingGlass} />
                         </PressableWithoutFeedback>
                     </Tooltip>
-                    <PressableWithoutFeedback
-                        accessibilityLabel={this.props.translate('sidebarScreen.buttonMySettings')}
-                        accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
-                        onPress={Session.checkIfActionIsAllowed(this.showSettingsPage)}
-                    >
-                        {Session.isAnonymousUser() ? (
-                            <View style={styles.signInButtonAvatar}>
-                                <Button
-                                    medium
-                                    success
-                                    text={this.props.translate('common.signIn')}
-                                    onPress={() => Session.signOutAndRedirectToSignIn()}
-                                />
-                            </View>
-                        ) : (
-                            <OfflineWithFeedback pendingAction={lodashGet(this.props.currentUserPersonalDetails, 'pendingFields.avatar', null)}>
-                                <AvatarWithIndicator
-                                    source={UserUtils.getAvatar(this.props.currentUserPersonalDetails.avatar, this.props.currentUserPersonalDetails.accountID)}
-                                    tooltipText={this.props.translate('common.settings')}
-                                />
-                            </OfflineWithFeedback>
-                        )}
-                    </PressableWithoutFeedback>
+                    <SignInOrAvatarWithOptionalStatus isCreateMenuOpen={this.props.isCreateMenuOpen} />
```

Comment:
@perunt could you elaborate from where does `this.props.isCreateMenuOpen` comes? Or is it always `undefined`?

Reply:
Hmm, tbh I didn't dig where it comes from but now I can see that it was undefined 
I see that it was removed (`isCreateMenuOpen` and `onAvatarClick`) without cleaning it inside the component
![image](https://github.com/Expensify/App/assets/48593211/75701faa-f105-4c6f-b97f-b422bce50e8a)

Reply:
Ok thanks for the input.